trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative './base_decorator'
+class TrimDecorator < Decorator
```

Comment:
- [ ] You also need to ensure the class name matches the one you defined. This should be `BaseDecorator`.

  ```
  class TrimDecorator < BaseDecorator
  ```

Reply:
@TedLivist Thanks! You helped me a lot. I really appreciate it!